packages/api-openfin/src/window.ts

```diff
@@ -192,14 +192,16 @@ export class Window extends Emitter implements ssf.Window {
           this.innerWindow = app.getWindow();
           this.id = `${this.innerWindow.uuid}:${this.innerWindow.name}`;
 
-          fin.desktop.InterApplicationBus.publish('ssf-new-window', {
-            windowName: this.innerWindow.uuid,
-            parentName: options.child ? Window.getCurrentWindow().innerWindow.uuid : null
+          Window.getCurrentWindow((win) => {
```

Comment:
Does this change introduce some potential latency on publishing the new window event? Previously we'd only be waiting on the current window if we were creating a child window, while now we always need the callback to be fired.
Would it make sense to grab it from the `fin.desktop.Window.getCurrent()` call we're making above to get the window options?